src/tree/node.rs

```diff
@@ -68,8 +68,13 @@ impl From<NodeId> for DefaultKey {
 pub(crate) struct NodeData {
     /// The layout strategy used by this node
     pub(crate) style: Style,
-    /// The results of the layout computation
-    pub(crate) layout: Layout,
+
+    /// The always unrounded results of the layout computation. We must store this separately from the rounded
+    /// layout to avoid errors from rounding already-rounded values. See <https://github.com/DioxusLabs/taffy/issues/501>.
+    pub(crate) unrounded_layout: Layout,
+
+    /// The final (possibly rounded) results of the layout computation
```

Comment:
"Possibly rounded"? In which cases would this not be rounded?
I think that would be useful to add to the comment

Reply:
They would not be rounded if the `use_rounding` setting is set to `false` using the `disable_rounding` method on the `Taffy` struct. I've added a comment :)